app/src/sharedTest/java/org/oppia/android/app/player/state/StateFragmentTest.kt

```diff
@@ -253,7 +257,10 @@ class StateFragmentTest {
     launchForExploration(TEST_EXPLORATION_ID_2).use {
       startPlayingExploration()
 
-      scrollToViewType(CONTINUE_INTERACTION)
```

Comment:
I think let's invert this refactor--scrollToViewType can use RecyclerViewScrollingActions' scrollToViewType (the current syntax is much cleaner so we actually lose that by refactoring at the test level).

Reply:
Done.

Reply:
This change will be handled in a separate PR, as migration will be handled independently.